invenio_vocabularies/contrib/names/datastreams.py

```diff
@@ -60,9 +61,11 @@ def _process_lambda_file(self, fileobj):
         """
         date_format = "YYYY-MM-DD HH:mm:ss.SSSSSS"
         date_format_no_millis = "YYYY-MM-DD HH:mm:ss"
-
+        time_shift = current_app.config["VOCABULARIES_ORCID_SYNC_SINCE"]
+        if self.since:
+            time_shift = self.since
         last_sync = arrow.now().shift(
-            days=-current_app.config["VOCABULARIES_ORCID_SYNC_DAYS"]
+            **{key: -value for key, value in time_shift.items()}
```

Comment:
Instead of `arrow.shift()`, could we use something like:


```python
since = datetime.utcnow() - timedelta(**current_app.config["VOCABULARIES_ORCID_SYNC_SINCE"])
```

Reply:
I was using this at the beginning and I don't have a personal preference, but there was a suggestion above that we agreed to use arrow. Following the suggestion I have updated it, see comment here: https://github.com/inveniosoftware/invenio-vocabularies/pull/366#discussion_r1686392070

Reply:
Updated! We can use arrow and timedelta, avoiding shift. Thanks!